src/pymonzo/transactions/schemas.py

```diff
@@ -94,7 +94,7 @@ class MonzoTransactionMerchant(BaseModel):
     online: bool
     atm: bool
     disable_feedback: bool
-    suggested_tags: str
+    suggested_tags: Optional[str] = None
```

Comment:
Do you mind putting this below `metadata` please, so the attributes with default values are at the end? Thanks!